heartbeat/monitors/active/tcp/tcp_test.go

```diff
@@ -18,22 +18,19 @@
 package tcp
 
 import (
-	"crypto/x509"
 	"fmt"
 	"net"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
-	"os"
 	"strconv"
 	"testing"
-	"time"
+
+	"github.com/elastic/go-lookslike/validator"
```

Comment:
nit: please sort imports

Reply:
++, I reset my IDE and lost my auto-sorter.